repos/fdbt-site/src/pages/api/exports.ts

```diff
@@ -0,0 +1,29 @@
+import { NextApiResponse } from 'next';
+import { getAndValidateNoc, redirectTo } from '../../utils/apiUtils';
+import { NextApiRequestWithSession } from '../../interfaces';
+import { getS3Exports } from '../../data/s3';
+import { getAllProducts } from '../../data/auroradb';
+import { triggerExport } from '../../utils/apiUtils/export';
+
+export default async (req: NextApiRequestWithSession, res: NextApiResponse): Promise<void> => {
+    const noc = getAndValidateNoc(req, res);
+    const products = await getAllProducts(noc);
+
+    const parts = new Date().toISOString().split('T');
+    const dateStr = parts[0] + '_' + parts[1].substr(0, 6).replace(/:/g, '');
```

Comment:
Could we rename these 2 variables? Finding it a little hard to understand what theyre doing, was thinking a more verbose name might help?

Reply:
👍